.github/workflows/ci-rootless-podman.yml

```diff
@@ -0,0 +1,51 @@
+name: CI-Podman-Rootless
+
+on:
+  pull_request: {}
+  push: { branches: [ main ] }
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Uninstall unwanted packages
+        run: sudo apt-get -q -y --purge remove podman moby-engine moby-buildx && sudo rm -rf /var/run/docker.sock
+      - name: Set XDG_RUNTIME_DIR
+        run: echo "XDG_RUNTIME_DIR=/run/user/$UID" >> $GITHUB_ENV
+      - name: Configure podman
+        run: |
+          mkdir -p $HOME/.config/containers
+          echo 'unqualified-search-registries = ["docker.io"]' > $HOME/.config/containers/registries.conf
+          echo ''                                             >> $HOME/.config/containers/registries.conf
+          echo '[[registry]]'                                 >> $HOME/.config/containers/registries.conf
+          echo 'prefix = "localhost"'                         >> $HOME/.config/containers/registries.conf
+          echo 'location = "localhost"'                       >> $HOME/.config/containers/registries.conf
+          echo 'insecure = true'                              >> $HOME/.config/containers/registries.conf
+      - name: Install latest podman release
+        # see https://podman.io/getting-started/installation#ubuntu
+        run: |
+          sudo mkdir -p /etc/apt/keyrings
+          curl -fsSL https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/Release.key \
+            | gpg --dearmor \
+            | sudo tee /etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg > /dev/null
+          echo \
+            "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg]\
+              https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/ /" \
+            | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:unstable.list > /dev/null
+          sudo apt-get update -qq
+          sudo apt-get -qq -y install podman
+      - name: Print podman environment information
+        run: podman info
+      - name: Enable podman socket
+        run: systemctl --user enable --now podman.socket
```

Comment:
We would like to simplify the action. Something similar to https://github.com/testcontainers/testcontainers-node/blob/64fd0420f30d290a4e26960db345c6f420c29da8/.github/workflows/test.yml#L70-L79 I wonder if there is any specific config in this setup which makes tc-java works with Podman.

Reply:
The installation is almost the same, only the formatting is different. I can copy the installation step you linked and remove the debug steps (running podman info and uploading the test report on failure).

The "Configure podman" step is needed for two things. To allow pulling from insecure registries (localhost), and to pull images from docker.io by default (this would not be needed for fully qualified image names, e.g. `docker.io/library/alpine` instead of `alpine`).

Reply:
I trimmed the action down a bit in c8af85e